python/ray/services.py

```diff
@@ -1253,6 +1280,11 @@ def start_ray_processes(address_info=None,
                       stderr_file=monitor_stderr_file,
                       cleanup=cleanup,
                       autoscaling_config=autoscaling_config)
+        if use_raylet:
+            start_raylet_monitor(redis_address,
+                                 stdout_file=monitor_stdout_file,
+                                 stderr_file=monitor_stderr_file,
```

Comment:
do we want these to go to the same log files as `monitor.py` or to their own log file?

Reply:
I think it makes sense to go to the same file as `monitor.py`, since only one of the monitors should ever be active at a given time (except for the autoscaler part of `monitor.py`).